src/views/AddItem.jsx

```diff
@@ -1,3 +1,93 @@
-export function AddItem() {
-	return <p>Hello from the <code>/add-item</code> page!</p>
+import { addItem } from '../api/firebase';
+import { useState } from 'react';
+
+export function AddItem({ listId }) {
+	// declare itemName, daysUntilNextPurchase, and radioSelect
+	// into separate state variables
+	const [itemName, setItemName] = useState('');
+
+	const [daysUntilNextPurchase, setDaysUntilNextPurchase] = useState(null);
+
+	const [radioSelect, setRadioSelect] = useState('soon');
+
+	// Store values inside itemData and sent over to database
+	// otherwise log an error if request fails
+	const onFormSubmit = (event) => {
+		event.preventDefault();
+		const itemData = {
+			itemName,
+			daysUntilNextPurchase,
+		};
+		try {
+			addItem(listId, itemData);
+			alert(`${itemData.itemName} was added to your list`);
+		} catch (error) {
+			console.log(error);
+		}
+	};
+
+	// Get user input and set itemName
+	const handleItemName = (event) => {
+		setItemName(event.target.value);
+	};
+
+	// Get radio selection and set daysUntilNextPurchase
+	const handlePurchaseDate = (event) => {
+		let date = null;
+		if (event.target.value === 'soon') {
+			date = 7;
+		} else if (event.target.value === 'kind-of-soon') {
+			date = 14;
+		} else if (event.target.value === 'not-soon') {
+			date = 30;
+		}
+		setDaysUntilNextPurchase(date);
+		setRadioSelect(event.target.value);
+	};
+
```

Comment:
(Mentors, please let me know if this isn't the right type of feedback for a code review!) This may be a preference thing, but I wonder about using the ternary operator instead of if else statements? I think here it may help lines 37-42 look a little cleaner (and/or wondering if this is something we norm as a group so that we're consistent? We also used an if statement in our work this week, so we could also change that if needed). I did a search for how to do ternary operator with multiple conditions and it would look something like this:
	`event.target.value === 'soon' ? date = 7`
			`: event.target.value === 'kind-of-soon' ? date = 14`
			`: date = 30;`
I could also be off with this, so take with a grain of salt :)

Reply:
@sarahmjenkins with three states, it probably does just come down to preference. Many more than three and ternaries get unwieldy IMHO. Typically I see it a little bit more like this:
```javascript
const value = event.target.value;
const date = value === 'soon' ? 7
  : value === 'kind-of-soon' ? 14
  : 30;
```

Another take might be:
```javascript
const map = {
  "soon": 7,
  "kind-of-soon": 14,
  "not-soon": 30
};
const date = map[event.target.value];
```

Reply:
I think a switch statement is also an alternative! Randall and I were just used to seeing if-statements, so we opted for that. But great points, ya'll! These suggestions look so much cleaner for sure.

Reply:
One thing to consider is how easily your code can be extended. `If` statements/ternary operators are fine for a limited number of scenarios, but what if there are new date requirements down the road? I think @mtfoley 's suggestion of adding a map is a great approach in this case.

Reply:
Agree. Readability and scalability should take precedence over one's preference.